### ISSUE TITLE
MD042 flagging valid MkDocs links as empty links

### ISSUE THREAD

**Issue by kattni** [rvben/rumdl#97]
Hello!

I reenabled this rule and it's actually weirder than I initially thought. 

This is a valid link in my MkDocs Markdown file:

```
[`str`][]
```

`rumdl check` is flagging this:

```
[MD042] Empty link found: [`str`][`str`] [*]
```

I verified in multiple flagged files that the links look like the original, and are not in fact duplicated into both sets of brackets.  This issue is also causing MD052 to flag:

```
Reference `str` not found
```

Not sure what to think of this one. Hopefully you can identify the issue. Let me know if I can help. I don't have any more information other than what I've shared here, but I'm happy to test anything you come up with.

**Comment by rvben** [comment]
Hi @kattni,

Thanks for bringing this up. This one (and similar ones) were indeed not being parsed correctly, and this will be fixed in the upcoming release. 

Thanks for helping bringing mkdocs support to the next level.

**Comment by rvben** [comment]
Fix released as part of 0.0.149.
